src/diffusers/models/attention.py

```diff
@@ -297,9 +365,12 @@ def __init__(
         dropout: float = 0.0,
         activation_fn: str = "geglu",
         final_dropout: bool = False,
+        inner_dim=None,
```

Comment:
Can't you leverage `mult` to define inner dim? Or are the numbers that uneven here?

Reply:
They're 2816 and 1024. So we could theoretically pass 2.75 as the mult and then cast to an int, but that feels a bit iffy. I'm ok either way though I think it's not a big issue if so

Reply:
Yeah ok with using `inner_dim` here